deepchecks/vision/checks/distribution/new_labels.py

```diff
@@ -0,0 +1,204 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains Train Test label Drift check."""
+import string
+from collections import defaultdict
+from secrets import choice
+from typing import Dict
+
+import torch
+
+from deepchecks.core import ConditionResult
+from deepchecks.core import DatasetKind, CheckResult
+from deepchecks.core.condition import ConditionCategory
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.utils.strings import format_percent, format_number
+from deepchecks.vision import Context, TrainTestCheck, Batch, VisionData
+from deepchecks.vision.utils.image_functions import prepare_thumbnail, draw_bboxes
+
+__all__ = ['NewLabels']
+
+from deepchecks.vision.vision_data import TaskType
+
+THUMBNAIL_SIZE = (200, 200)
+
+
+def draw_image(data: VisionData, sample_index: int, class_id: int) -> str:
```

Comment:
That looks pretty generic to me. @matanper @yromanyshyn any place or way we can make stuff like this a common util?

Reply:
it's not generic enough, for example:
here filtering specific class_id for the bboxes.
in another place not showing bboxes at all,
in third place (outliers) filerting only single bbox by it's index in the list of bboxes